services/QuillLMS/app/models/canvas_account.rb

```diff
@@ -24,4 +24,12 @@
 class CanvasAccount < ApplicationRecord
   belongs_to :canvas_instance
   belongs_to :user
+
+  def self.canvas_id(canvas_instance_id, canvas_user_external_id)
```

Comment:
Do you need this method anymore? Not sure if it's used in the other PR now.

Reply:
Good point.